### PR TITLE
feat: support concurrent safe

### PR DESCRIPTION
**Pull request by childrentime**

### CODE REVIEW

**Comment by childrentime** [comment]
@FleetAdmiralJakob Hey, if you're interested, you could take a look at this PR.

File: packages/core/src/useDarkMode/index.ts
Comment (FleetAdmiralJakob) [inline packages/core/src/useDarkMode/index.ts:1]: lgtm

File: packages/core/src/useActiveElement/index.ts
Comment (FleetAdmiralJakob) [inline packages/core/src/useActiveElement/index.ts:1]: lgtm

File: packages/core/src/utils/browser.ts
Comment (FleetAdmiralJakob) [inline packages/core/src/utils/browser.ts:1]: lgtm

File: packages/core/src/useMouse/index.ts
Comment (FleetAdmiralJakob) [inline packages/core/src/useMouse/index.ts:1]: lgtm

File: packages/core/src/useLatest/index.ts
Comment (FleetAdmiralJakob) [inline packages/core/src/useLatest/index.ts:1]: lgtm

**Review by FleetAdmiralJakob — APPROVED** [review]
lgtm, although im not a react wizard. dont see why this shouldn't work haha

**Pull request merged**